src/Components/Aspire.Elastic.Clients.Elasticsearch/ElasticsearchHealthCheck.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Elastic.Clients.Elasticsearch;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+// TODO: Use health check from AspNetCore.Diagnostics.HealthChecks once following PR released:
+// https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/2244
```

Comment:
Can we open an issue to track that this needs to be done, if the nuget isn't available yet?